scripts/core/INTRO.rst

```diff
@@ -404,6 +404,31 @@ Specific environment variables can be set to control the behavior of unified run
 
     This environment variable is default enabled on Linux, but default disabled on Windows.
 
+CTS Environment Variables
+-------------------------
+
+The following environment variables are used by the CTS runner and can be used to specify the platform that the test
+framework should run on. This can be used during development and testing to run CTS tests in case multiple platforms
+are available. If both filters are specified, then they both must match a platform for it to be selected. If there are
+no valid platforms, then the tests will fail. Command line arguments take priority over these variables.
+
+.. envvar:: UR_CTS_ADAPTER_PLATFORM
+
+    A specifier list in the form of `[(backend):](platform name)[;[(backend)]:(platform name)]...`. If a backend
+    specific specifier is present in the list and the test is running for that backend, the device with the given name
+    is chosen. Otherwise, it must match the name of the specifier from the list with no backend.
+
+    For example, if the test device has multiple platforms and you want to run tests on the "ABC Corp" backend when
+    testing OpenCL and "XYZ Org" when testing level zero, you'd use `OPENCL:ABC Corp;LEVEL_ZERO:XYZ Org`. This form is
```

Comment:
Are the backend and platform matches case insensitive as well?

Reply:
Backends are not, but the platform must match exactly. Updated the docs.

Reply:
I'd like to avoid ALL_CAPS in the backends please, or at least be consistent with `UR_CTS_BACKEND` and make it case insensitive.

Reply:
... I completely misread that. Backends are indeed case insensitive.